stock_deposit/README.rst

```diff
@@ -0,0 +1,92 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=============
+Stock deposit
+=============
+
+This module extends standard WMS to manage customers deposit locations by
+warehouse.
+Deposited stocks can't be selected in regular delivery orders unless you have
+been returned it.
+Deposited stocks have been evaluated in inventory valuation.
+
+Installation
+============
+
+The installation module process create a deposit location per warehouse and
+two picking types, one for outgoing deposits and other for returned deposits.
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+#. Go to *Inventory > Settings > Product Owners* and click
+   "**Manage consignee stocks (advanced)**".
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to products and create one of type "Stockable".
+#. Update quantities on hand to have stock of it.
+#. Go to inventory dashboard and click on "Outgoing deposits" card to do a new
+   transfer.
+#. Create a picking with owner and select the product to do the transfer.
+
+To track current deposits:
+
+#. After that process, you can view deposit quantities in product form view.
+#. You can also see the deposits for all your products in the menu entry
+   Inventory --> Reports --> Deposit location inventory.
+
+For regularizing deposited quantities (this means to deliver deposited stock
+to the customer as a normal outgoing move.), you need:
+
+#. Go to deposit inventory, select quants that you want to regularize and click
```

Comment:
If here is s/deposit inventory/Deposited Inventory Valuation , where is the report? I not, I cannot find this menú item.

![2016-10-19_21-39-23](https://cloud.githubusercontent.com/assets/8736623/19534552/883e0436-9644-11e6-8df3-6a6c2b9246ee.png)